packages/components/src/SquiggleEditor.tsx

```diff
@@ -46,27 +46,16 @@ export class SquiggleEditor extends React.Component<
     let props = this.props;
     return (
       <div>
-        <ReactCodeJar
```

Comment:
Did you remove the ReactCodeJar from the package.json? (Can't find that line here)

Reply:
No, I've done that now though